### PR TITLE
(#17) strong list

### PR DESCRIPTION
**Pull request by klepas**
Implements the repeating word style. I’ve decided to rename this for the purpose we present. This might be an issue, esp. when used for another context, but hey… it’s better than our current naming style (which is already diff. to what the class used to called — “repeating word style” vs `.highlighted-list`).

@dominikwilkowski Do we want to have a quick pow-wow with Andrew and Jools about the naming here, or leave it?

This also includes a bugfix for headings, where I listed the leading as being 1.4, without it being 1.4. The `.display-[1…6]` headings now receive a line-height of `$uikit-leading * 0.875` (which curently becomes 1.4).

### CODE REVIEW

File: packages/department-list/src/sass/_module.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+.uikit-department-list {
+
+	.uikit-body & > li {
+		list-style: none;
+		margin-left: 0;
+		margin-bottom: uikit-space( 1 );
+	}
+
+	a {
```
Comment (dominikwilkowski) [inline packages/department-list/src/sass/_module.scss]: This is not specific enough. Should be `> a`

File: packages/department-list/src/sass/_module.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+@import '_globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Contents links style
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+.uikit-department-list {
+
+	.uikit-body & > li {
```
Comment (dominikwilkowski) [inline packages/department-list/src/sass/_module.scss]: you don't need the `&` here

File: packages/department-list/src/sass/_module.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		margin-left: 0;
+		margin-bottom: uikit-space( 1 );
+	}
+
+	a {
+		font-size: 28px;
+		font-weight: bold;
+		line-height: ($uikit-leading * 0.875);
+
+		> span {
```
Comment (dominikwilkowski) [inline packages/department-list/src/sass/_module.scss]: Span should have a class instead

File: packages/department-list/src/sass/_module.scss (hunk 1 above)
Comment (dominikwilkowski) [inline packages/department-list/src/sass/_module.scss]: Also make it a class

File: packages/department-list/tests/site/test.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+//dependencies
+@import '../../../core/lib/sass/_module.scss';
+@import '../../../body/lib/sass/_module.scss'; //dependency via link-lists
```
Comment (dominikwilkowski) [inline packages/department-list/tests/site/test.scss]: Can you put it into testing then?

File: packages/headings/src/sass/_module.scss [hunk 5]
```diff
@@ -26,7 +26,7 @@
 	.uikit-display-#{$i} {
 		font-family: $uikit-font;
 		display: block;
-		line-height: $uikit-leading; //1.142857143;//1.4 from 16 (default: core)
+		line-height: ($uikit-leading * 0.875); //1.4 from 16 (default: core)
```
Comment (dominikwilkowski) [inline packages/headings/src/sass/_module.scss]: I'm concerned about these magic numbers...

File: packages/department-list/src/sass/_module.scss (hunk 1 above)
Comment (klepas) [inline packages/department-list/src/sass/_module.scss]: Thanks — will do.

File: packages/department-list/src/sass/_module.scss (hunk 3 above)
Comment (klepas) [inline packages/department-list/src/sass/_module.scss]: Good point.

File: packages/headings/src/sass/_module.scss (hunk 5 above)
Comment (klepas) [inline packages/headings/src/sass/_module.scss]: Can we make a `$uikit-leading--heading` var then?

Comment (dominikwilkowski) [inline packages/headings/src/sass/_module.scss]: no that's just yet another var to keep in your mental model. No we need a better way of converting them. Ideally we should have something like: `leading(1)` `leading(2)` `leading(3)` `leading(4)` etc... or we have a mixin that takes the font-size and calculates the line height from there. We can curate the line height in there too if we feel certain font sizes should have different edge cases.

**Comment by klepas** [comment]
Re naming: I’m then tempted to name it the same as we’ve had so far: `.highlighted-list`, but I’ll ask Jools.

File: packages/department-list/tests/site/test.scss (hunk 4 above)
Comment (klepas) [inline packages/department-list/tests/site/test.scss]: @dominikwilkowski why for? It’s a dependency, no?

File: packages/department-list/src/sass/_module.scss (hunk 2 above)
Comment (klepas) [inline packages/department-list/src/sass/_module.scss]: @dominikwilkowski Sure? I think I do, to unset styles that are being otherwise inherited from `link-lists` (the margins)?

Comment (dominikwilkowski) [inline packages/department-list/src/sass/_module.scss]: right. All good.

File: packages/department-list/tests/site/test.scss (hunk 4 above)
Comment (dominikwilkowski) [inline packages/department-list/tests/site/test.scss]: not a direct one.

File: packages/headings/src/sass/_module.scss (hunk 5 above)
Comment (klepas) [inline packages/headings/src/sass/_module.scss]: I’m happy to go down either path. A `leading()` function would be super easy to do — I’m happy to pick this up and make this edit if you like @dominikwilkowski ?

**Comment by klepas** [comment]
Just documenting i’ve settled on `strong-list` for now, and we can revisit this naming stuff on Wednesday together.

**Comment by dominikwilkowski** [comment]
Please merge `develop` into this branch to enable the preview.

**Comment by dominikwilkowski** [comment]
merged

**Comment by dominikwilkowski** [comment]
staging: https://65-78596482-gh.circle-artifacts.com/0/tmp/circle-artifacts.ziSNoZ5/packages/strong-list/tests/site/index.html
